react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+class CustomButton extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {count: false};
+        this.handleClick = this.handleClick.bind(this);
+    }
+
+    
+
+    handleClick() {
+        this.setState({count: this.state.count + 1})
+    }
+
+    render() {
+        let buttonClass = 'hot';
+
+    if(this.state.count >= 18){
+        buttonClass = 'hot6';
+    }
+    if(this.state.count >= 15){
+        buttonClass = 'hot5';
+    }
+    if(this.state.count >= 12){
+        buttonClass = 'hot4';
+    }
+    if(this.state.count >= 9){
+        buttonClass = 'hot3';
+    }
+    if(this.state.count >= 6){
+        buttonClass = 'hot2';
+    }
+    if(this.state.count >= 3){
+        buttonClass = 'hot1'
+        
+    }

```

Comment:
Here you should be using `else if`. As it is now, once it is over three clicks the class will always be `hot1`